src/reducer.js

```diff
@@ -0,0 +1,26 @@
+const initState = [];
+const rocketsReducer = (state = initState, action) => {
+  if (action.type === 'FETCH_ROCKETS') {
+    return action.payload;
+  }
+  if (action.type === 'RESERVE_ROCKET') {
+    state.map((elem) => {
+      if (elem.id === action.payload) {
+        elem.reserved = true;// eslint-disable-line no-param-reassign
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is quality and clean.